ios/brave-ios/Sources/Brave/WebFilters/AdBlock/AdBlockGroupsManager.swift

```diff
@@ -302,7 +302,18 @@ import os
   func compileEngineIfFilesAreReady(for engineType: GroupedAdBlockEngine.EngineType) {
     let manager = self.getManager(for: engineType)
     let enabledSources = sourceProvider.enabledSources(for: engineType)
-    guard manager.checkHasAllInfo(for: enabledSources) else { return }
+    let availableSources = manager.compilableFiles(for: enabledSources)
+      .map({ $0.filterListInfo.source })
+    guard enabledSources.allSatisfy({ availableSources.contains($0) }) else {
```

Comment:
This loop[ is still running even it is not debug and we dont need logs

Reply:
I really need logs to test these features out and make sure things are working properly. I missed this mistake and i think i would have spotted it if I had this log. Though I can remove it if you insist.

Reply:
Actually just removed it.